test/In.ProjectEKA.DefaultHipTest/Link/Builder/TestBuilder.cs

```diff
@@ -0,0 +1,24 @@
+using Bogus;
+using In.ProjectEKA.DefaultHip.Link;
+using Microsoft.Extensions.Configuration;
+
+namespace In.ProjectEKA.DefaultHipTest.Link.Builder
+{
+    public class TestBuilder
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4807294)